### PR TITLE
Issue 12

### PR DESCRIPTION
**Pull request by matentzn**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,73 +2,143 @@ MAKEFLAGS 				+= --warn-undefined-variables
...
+	echo prepare-inputs $*
 
+.PHONY: prepare-corpus1
+prepare-corpus1: corpora/corpus1/scrambled1/corpus.yml
+	$(MAKE) SCRAMBLE_FACTOR=0.5 corpora/corpus1/scrambled-r0.5/corpus.yml
 
-.PHONY: shuffle-semsim
+.PHONY: run-corpus1
+run-corpus1:
+	$(MAKE) SCRAMBLE_FACTOR=1 results/exomiser-13.01/default-corpus1-scrambled1/results.yml
```
Comment (matentzn) [inline Makefile:136]: Why is the scramble factor necessary as a variable?

File: Makefile [hunk 2]
```diff
@@ -2,73 +2,143 @@ MAKEFLAGS 				+= --warn-undefined-variables
...
-	$(eval TERM=$(shell bash -c "cat $(RAW_DATA_FOLDER)/random-hp-terms.txt"))
-	$(eval TERM2=$(shell bash -c "cat $(RAW_DATA_FOLDER)/random-mp-terms.txt"))
-	runoak -i $< similarity -p i,p $(TERM) @ $(TERM2) -O csv -o $@
+.PHONY: prepare-inputs1
+prepare-inputs1: configurations/exomiser-13.01/default/config.yml
+	echo prepare-inputs $*
 
+.PHONY: prepare-corpus1
+prepare-corpus1: corpora/corpus1/scrambled1/corpus.yml
+	$(MAKE) SCRAMBLE_FACTOR=0.5 corpora/corpus1/scrambled-r0.5/corpus.yml
```
Comment (matentzn) [inline Makefile:132]: Sweet!!

File: Makefile [hunk 3]
```diff
@@ -2,73 +2,143 @@ MAKEFLAGS 				+= --warn-undefined-variables
...
+results/exomiser-13.01/default-corpus1-scrambled1/results.yml: configurations/exomiser-13.01/default/config.yml corpora/corpus1/scrambled1/corpus.yml
+	mkdir -p $(shell dirname $@)
+	pheval run --input-dir $(shell pwd)/configurations/exomiser-13.01/default --testdata-dir $(shell pwd)/corpora/corpus1/scrambled1 --runner exomiserphevalrunner --tmp-dir $(shell pwd)/$(TMP_DATA)/ --output-dir $(shell pwd)/$(shell dirname $@) --config $(shell pwd)/$<
+	touch $@
+
+corpora/corpus1/scrambled1/corpus.yml: $(TEST_DATA)/template_vcf/template_exome_hg19.vcf.gz
+	test -d $(shell dirname $@)/vcf || mkdir -p $(shell dirname $@)/vcf
+	test -d $(shell dirname $@)/phenopackets || mkdir -p $(shell dirname $@)/phenopackets
+	test -L $(shell pwd)/corpora/corpus1/scrambled1/template_exome_hg19.vcf.gz || ln -s $(shell pwd)/$< $(shell dirname $@)/vcf/
+	pheval-utils create-spiked-vcfs --template-vcf-path $(shell dirname $@)/vcf/template_exome_hg19.vcf.gz --phenopacket-dir=$(TEST_DATA)/phenopackets/single --output-dir $(shell dirname $@)/vcf
```
Comment (matentzn) [inline Makefile:104]: Correct spirit but I think we should avoid variables for now (test data) best have this goal depend on the default dir with the original corpus information?

File: Makefile [hunk 4]
```diff
@@ -2,73 +2,143 @@ MAKEFLAGS 				+= --warn-undefined-variables
...
+	test -d $(EXOMISER_DATA_FOLDER)/hp-mp2-semsim.scrambled1 || cp -rf configurations/exomiser-13.01/default/2209_phenotype/ $(EXOMISER_DATA_FOLDER)/hp-mp2-semsim.scrambled1
+	java -cp $(H2_JAR) org.h2.tools.RunScript -user sa -url jdbc:h2:/$(EXOMISER_DATA_FOLDER)/hp-mp2-semsim.scrambled1/2209_phenotype -script configurations/semsim1/hp-mp2.semsim.scrambled1.sql
+
+results/phen2gene/default-corpus1-scrambled1/results.yml: configurations/exomiser-13.01/default/config.yml
+	pheval run --input-dir $(PHEN2GENE_LIB) --testdata-dir $(shell pwd)/corpora/corpus1/scrambled1 --runner phen2genephevalrunner --tmp-dir $(shell pwd)/$(TMP_DATA)/ --output-dir $(shell dirname pwd)/$(shell dirname $@) --config $<
+	touch $@
+
+results/exomiser-13.01/default-corpus1-scrambled1/results.yml: configurations/exomiser-13.01/default/config.yml corpora/corpus1/scrambled1/corpus.yml
+	mkdir -p $(shell dirname $@)
+	pheval run --input-dir $(shell pwd)/configurations/exomiser-13.01/default --testdata-dir $(shell pwd)/corpora/corpus1/scrambled1 --runner exomiserphevalrunner --tmp-dir $(shell pwd)/$(TMP_DATA)/ --output-dir $(shell pwd)/$(shell dirname $@) --config $(shell pwd)/$<
```
Comment (matentzn) [inline Makefile:97]: Fantastic!

**Review by matentzn — COMMENTED** [review]
Left some comments feel free to contradict me! Look great so far!!

**Comment by souzadevinicius** [comment]
Closed by #126

**Pull request closed without merging**